scripts/privacyidea

```diff
@@ -988,7 +988,7 @@ def create_arguments():
                           "of length",
                           metavar="NUMBER",
                           type=int,
-                          default=0)
+                          default=6)
```

Comment:
I would like to keep this a zero.
Because my usual usecase is enrolling the yubikey in OATH mode without a prefix.
In this case I guess I would get an OATH mode (default) with a prefix of 6 (default) which is really uncommon.
I.e. there would be an uncommon default combination.